### ISSUE TITLE
Provide configurations for eslint/prettier/... in dev tips 

### ISSUE THREAD

**Issue by NavyAdmiral** [smartcontractkit/chainlink#544]
Hi. I tried to configure prettier/eslint with standardjs options but I couldn't get it to produce results consistent with the codebase. Can we add tips on how to achieve this with available formating tools?

Unfortunately, after my Prettier formats the code, `yarn lint` isn't enough to keep the commit clean from unnecessary additions/deletions.

cc @rupurt

**Comment by rupurt** [comment]
@NavyAdmiral what OS/editor combination are you using?

**Comment by rupurt** [comment]
Have you tried the standardjs extension? https://marketplace.visualstudio.com/items?itemName=chenxsan.vscode-standardjs

**Comment by NavyAdmiral** [comment]
no bueno

**Comment by rupurt** [comment]
Can you provide an example of a change that didn't pass standardjs checks?

**Comment by rupurt** [comment]
@NavyAdmiral I've just installed VS Code on OS X and installed the extension above. It produces linting errors as I type. 

<img width="870" alt="screen shot 2018-09-05 at 8 51 23 am" src="https://user-images.githubusercontent.com/680789/45105376-f25ecb00-b0e8-11e8-8320-8d96d59ecefd.png">

You will need to fix them manually by default. If you want to use the `autofix` feature you will need to disable autosave.

```
{
    "files.autoSave": "off",
    "standard.autoFixOnSave": true
}
```

**Comment by NavyAdmiral** [comment]
@rupurt Well, I did try that, but it's a bit lenient on formatting. E.g:
```js
import { jobSpecSelector, jobRunsSelector, jobRunsCountSelector } from 'selectors'
```
For this code, it won't format it to:
```js
import {
  jobSpecSelector,
  jobRunsSelector,
  jobRunsCountSelector
} from 'selectors'
```
So multiline import is what's currently in the repo, but if I had to write one inline, I would expect the formatter to format it according to our preferences.
Another example:
```js
                <Button
                  variant='outlined'
                  color='primary'
                  disabled={fetching}
                  onClick={handleClick}>
                  Run
                </Button>
```
After writing this code ^, I would expect the formatter to inline it to `                <Button variant='outlined' color='primary' disabled={fetching} onClick={handleClick}>`, because inline is consistent with the repo. And there are other examples where standardjs formatter will not format exactly right. I think these could have been easily configured with eslintrc, but I don't know about it can be done with standardjs.
